website/static/worker.js

```diff
@@ -26,7 +26,17 @@ self.require = function require(path) {
   if (path === "./third-party") {
     return {};
   }
-  return self[path.replace(/.+-/, "")];
+
+  if (~path.indexOf("parser-")) {
+    var parser = path.replace(/.+-/, "");
+    if (!parsersLoaded[parser]) {
+      importScripts("lib/parser-" + parser + ".js");
+      parsersLoaded[parser] = true;
+    }
+    return self[parser];
```

Comment:
Is this actually defined? If it works it's a good solution

Reply:
As I understand the previous code, that's exactly how it was working before. We called `importScripts` below just for this to work `return self[path.replace(/.+-/, "")]`

Reply:
It's a bit hackish, since it requires knowledge of the `moduleName` in the rollup config